### ISSUE TITLE
hbond with image: got 0 

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#105]
I was trying to use image on/off but got 0 hbond with image on

**case 1**
```bash
parm tz2.ortho.parm7
trajin tz2.ortho.nc
hbond out hb.dat
```

output
```bash

#Frame   HB_00001[UU]
       1            5
       2            5
       3            5
       4            4
       5            4
       6            5
       7            6
       8            5
       9            5
      10            4
```

**case 2**
```bash
parm tz2.ortho.parm7
trajin tz2.ortho.nc
hbond image out hb.dat
```

output
```bash
#Frame   HB_00001[UU]
       1            0
       2            0
       3            0
       4            0
       5            0
       6            0
       7            0
       8            0
       9            0
      10            0
```

according to this topic http://archive.ambermd.org/201506/0455.html, is 'image' in hbond similiar to `autoimage`? if yes, why the output in two cases are so different?

**Comment by hainm** [comment]
full log for case 2

```

CPPTRAJ: Trajectory Analysis. V16.00b
    ___  ___  ___  ___
     | \/ | \/ | \/ |
    _|_/\_|_/\_|_/\_|_

| Date/time: 10/09/15 00:20:15
| Available memory: 508.012 MB

INPUT: Reading Input from file tz2.hbond.in
  [parm tz2.ortho.parm7]
        Reading 'tz2.ortho.parm7' as Amber Topology
  [trajin tz2.ortho.nc]
        Reading 'tz2.ortho.nc' as Amber NetCDF
  [hbond image out hb.dat]
  HBOND: Searching for Hbond donors/acceptors in region specified by *
        Distance cutoff = 3.000, Angle Cutoff = 135.000
        Writing # Hbond v time results to hb.dat
        Imaging enabled.
---------- RUN BEGIN -------------------------------------------------

PARAMETER FILES:
 0: 'tz2.ortho.parm7', 5293 atoms, 1704 res, box: Orthogonal, 1692 mol, 1691 solvent, 10 frames

INPUT TRAJECTORIES:
 0: 'tz2.ortho.nc' is a NetCDF AMBER trajectory, Parm tz2.ortho.parm7 (Orthogonal box) (reading 10 of 10)
  Coordinate processing will occur on 10 frames.
TIME: Run Initialization took 0.0000 seconds.

BEGIN TRAJECTORY PROCESSING:
.....................................................
ACTION SETUP FOR PARM 'tz2.ortho.parm7' (1 actions):
  0: [hbond image out hb.dat]
        Set up 38 acceptors:
        Set up 31 donors:
        Imaging on.
        Estimated max potential memory usage: 0.09 MB
----- tz2.ortho.nc (1-10, 1) -----
 0% 11% 22% 33% 44% 56% 67% 78% 89% 100% Complete.

Read 10 frames and processed 10 frames.
TIME: Trajectory processing: 0.0186 s
TIME: Avg. throughput= 537.2300 frames / second.

ACTION OUTPUT:
    HBOND: Actual memory usage is 0.00 MB
        0 solute-solute hydrogen bonds.

DATASETS:
  2 data sets:
        tz2.ortho.parm7 "tz2.ortho.parm7" (topology), size is 5293 'tz2.ortho.parm7', 5293 atoms, 1704 res, box: Orthogonal, 1692 mol, 1691 solvent
        HB_00001[UU] "HB_00001[UU]" (integer), size is 10

DATAFILES:
  hb.dat (Standard Data File):  HB_00001[UU]
---------- RUN END ---------------------------------------------------
TIME: Total execution time: 0.0673 seconds.
--------------------------------------------------------------------------------
To cite CPPTRAJ use:
Daniel R. Roe and Thomas E. Cheatham, III, "PTRAJ and CPPTRAJ: Software for
  Processing and Analysis of Molecular Dynamics Trajectory Data". J. Chem.
  Theory Comput., 2013, 9 (7), pp 3084-3095.

```

**Comment by drroe** [comment]
Fixed by #107, thanks for the report.
